quickstarts/redmine/config.yml

```diff
@@ -0,0 +1,45 @@
+
+
+slug: redmine
+description: |
+  ## Comprehensive monitoring quickstart for Redmine
+  We can monitor your Redmine application in a complex environment and keep you on top of performance with our Ruby agent.
+  ## Why monitor Redmine?
+  The performance monitoring tool for Redmine helps you monitor the entire infrastructure and identifies the critical issues in your Redmine application. Redmine performance monitoring enables you to get a complete overview of Redmine errors and know when and where your Redmine application slows down. Some key components are:
+  - Helps you identify issues that are truly important and quickly resolve them. Gain knowledge of performance issues in advance and thus giving you ample time to avoid application downtime.
```

Comment:
Make this two bullets:

"- Identify issues that are truly important and quickly resolve them.
- Be warned of performance issues ahead of time, giving you plenty of time to avoid downtime."